spec/flipper/middleware/memoizer_spec.rb

```diff
@@ -342,7 +342,7 @@
     it 'eagerly caches known features for duration of request' do
       memory = Flipper::Adapters::Memory.new
       logged_memory = Flipper::Adapters::OperationLogger.new(memory)
-      cache = ActiveSupport::Cache::DalliStore.new(ENV['MEMCACHED_URL'])
+      cache = ActiveSupport::Cache::MemoryStore.new
```

Comment:
The adapter being used here doesn't seem to matter, and since DalliStore is deprecated, I just swapped it with MemoryStory.